data/wanderer/wanderers start.txt

```diff
@@ -260,113 +260,57 @@ mission "Wanderers: Translation Machine"
-		has "human cultural data"
-	choice
-		`	"No problem, I'll bring you a copy of the database as soon as I can."`
-			accept
-		`	"How will I convince them to give me a copy of their data?"`
-	`	Your question seems to utterly confuse him. "Information," he says. "It is what you call a non-rival good, is it not? When I give information, I do not lose it. Sharing it costs me nothing. The librarians, surely they will understand this. Have no fear."`
-		accept
-	
-	label data
-	choice
-		`	"I actually already have a copy of that data."`
-	`	"You do? Please go get it!" he exclaims.`
-	`	You go to your ship to retrieve the data and give it to him upon returning. His eyes light up with excitement as he makes a copy of the card and returns the original to you.`
-		goto give
-	
-	label return
-	`You return to Eruk's workshop and give him the data card. His eyes light up with excitement as he makes a copy of the card and returns the original to you.`
-	
-	label give
-	`	"I have spent my whole life studying human culture," he says. "This is a treasure to me. And now, here is your speaking machine." He hands you a small, spherical device, with speakers and microphones mounted on it, as well as what look like camera lenses. "The device is programmed with the human and Hai languages," he says. "It is not always possible to translate perfectly between our two languages, but it will try to give you a sense of the range of meanings in what is being said."`
-	choice
-		`	"Thank you. I'm eager to try it out!"`
-			accept
-		`	"Is there anything else I need to know?"`
-	`	He humors you by demonstrating how the machine works, holding a conversation with you where he speaks in the Hai language, and the machine automatically translates for you and then translates your responses back into his language. It apparently uses its cameras to track who you are speaking to or listening to. It's an impressive piece of technology, and easy enough to use. You thank him for his help, and return to your ship.`
-		accept
-
-
-
-mission "Human Cultural Archives"
-	description "Eruk, the Hai technologist on <origin>, has agreed to give you a translation machine if you can somehow get him a copy of the human cultural data from <destination>."
+mission "Wanderers: Translation Algorithm Training Data"
+	description "Bring a copy of the human cultural archives to <destination> so the Wanderers can train an automated translation algorithm."
 	landing
-	source "Greenwater"
-	destination "Alexandria"
+	source "Alexandria"
+	destination "Vara K'chrai"
 	to offer
 		has "Wanderers: Translation Machine: done"
-		not "human cultural data"
 	on offer
-		conversation "eruk cultural data"
-	on visit
-		dialog
-			`The station has a small museum gift shop that sells a copy of the entire archive on a data card. It only costs 40 credits, but because you have done a horrible job at managing your finances you do not have even that much cash on hand right now.`
-			`	Go earn some money, then return here.`
```

Comment:
I really want to keep this scathing telling-off of the player.

Reply:
This is actually the critical `on visit` that I removed. Thank you for noticing that.

Reply:
Now I've noticed the old "on visit" text makes no sense, and the "on complete" makes no sense if you saw the "on visit" first. I will have to update both in a few hours once I have time.

Reply:
It took some doing, but I got it to work and tested both routes:

1. If you lack 40 credits, the visit message has the library search and scolds you for being bankrupt. Completion message congratulates you for paying 40 credits, and does not do the search.
2. If you have 40 credits, the completion message has the library search and you pay 40 credits to finish.